website/addons/s3/static/s3NodeConfig.js

```diff
@@ -91,31 +91,45 @@ ViewModel.prototype.updateBucketList = function(){
 
 ViewModel.prototype.selectBucket = function() {
     var self = this;
+
+    // The current version of boto does not handle bucket names containing periods
+    var isValidBucket = /^(?!.*(\.\.|-\.))[^.][a-z0-9\d-]{2,61}[^.]$/;
```

Comment:
`isValidBucket` is already defined elsewhere in this file. There should be only one of these.